File: src/test/java/aoc2024/Day01Test.java
package aoc2024;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

class Day01Test {

    @Test
    public void aoc_example_part1() {
        assertEquals("11", new Day01().asTest().solvePart1());
    }

    @Test
    public void aoc_example_part2() {
        assertEquals("31", new Day01().asTest().solvePart2());
    }
}

File: src/test/java/aoc2025/Day01Test.kt
package aoc2025

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

class Day01Test {

    @Test
    fun aoc_example_part1() {
        assertEquals("3", Day01().asTest().solvePart1())
    }

    @Test
    fun aoc_example_part2() {
        assertEquals("6", Day01().asTest().solvePart2())
    }
}
